CIP-1694/README.md

```diff
@@ -515,7 +518,7 @@ The following table details the ratification requirements for each governance ac
   The DRep vote threshold that must be met as a percentage of *active voting stake*.
 
 * **SPOs**<br/>
-  The SPO vote threshold which must be met as a percentage of the stake held by all stake pools.<br/>
+  The SPO vote threshold which must be met as a certain threshold of the total active voting stake, excepting Hard Fork Governance Actions. Due to the need for robust consensus around Hard Fork initiations, these votes must be met as a percentage of the stake held by all stake pools. <br/>
```

Comment:
Same here. To dispel confusion about what constitutes 'active voting stake,' it's important to note that the 'Auto-Abstain' voting option is typically not included in the 'active voting stake' calculations, while 'Auto-No-Confidence' is. Should we add some additional clarifications regarding the hard fork exception?

Reply:
The SPOs' relationship to the Automated Voting Options being added as a Note after the Automated Voting Options section is placed to dovetail off of the *existing* explanations of how these options affect the Active Voting Stake.